fury/ui/tests/test_elements.py

```diff
@@ -792,9 +792,9 @@ def test_ui_combobox_2d(interactive=False):
 
 
 def test_ui_draw_shape():
```

Comment:
this test is a bit too light for me.  You should use `snapshot` to save the scene and check if we have the expected color and the expected number of objects. 

You can look at this example to see how to do it: https://github.com/fury-gl/fury/blob/master/fury/tests/test_window.py#L26-L30